web-api/migration-terraform/main/lambdas/migration-segments.js

```diff
@@ -51,16 +54,21 @@ const migrateRecords = async ({
     items = await bugMigration0036(items);
   }
 
-  if (!ranMigrations['0036-phone-number-format.js']) {
+  if (!ranMigrations['migration0036.js']) {
```

Comment:
Same as above — I was expecting this to be reverted to `0036-phone-number-format.js`.